src/draw/renesas/dave2d/lv_draw_dave2d_image.c

```diff
@@ -112,6 +114,94 @@ static void img_draw_core(lv_draw_unit_t * u_base, const lv_draw_image_dsc_t * d
     src_blend_mode    = d2_getblendmodesrc(u->d2_handle);
     dst_blend_mode    = d2_getblendmodedst(u->d2_handle);
 
+#if defined(RENESAS_CORTEX_M85)
+#if (BSP_CFG_DCACHE_ENABLED)
+    d1_cacheblockflush(u->d2_handle, 0, src_buf,
+                       img_stride * decoder_dsc->header.h); //Stride is in bytes, not pixels/texels
+#endif
+#endif
+
+    if(LV_COLOR_FORMAT_RGB565A8 == cf) {
+
+        lv_point_t p1[4] = { //Points in clockwise order
+            {0, 0},
+            {decoder_dsc->header.w - 1, 0},
```

Comment:
We should use decoded->header instead. Same for below. The decoded image may differ with original image. For example, software renderer doesn't support I1/2/4/8, thus decoder will convert to ARGB8888.

Reply:
I have update the pull request, to use decoded->header